### PR TITLE
ticdc : async exec ddl doc update

### PR DESCRIPTION
**Pull request by sdojjy**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-ddl.md [hunk 1]
```diff
@@ -46,9 +46,18 @@ summary: 了解 TiCDC 支持同步的 DDL 和一些特殊情况
 
 ## DDL 同步注意事项
 
+### Add/Create index DDL 异步执行
+为了减小对 Chnagefeed 同步延迟的影响，TiCDC 会将 Add/Create index DDL 异步执行。这意味着，当 TiCDC 同步到 Add/Create index DDL 时，该 DDL 操作并不会立即执行，而是会在后台异步执行。
```
Comment (nongfushanquan) [inline ticdc/ticdc-ddl.md:50]: 为了减小对 Chnagefeed 同步延迟的影响，TiCDC 会将 Add/Create index DDL 异步执行，即 TiCDC 会将 Add/Create index DDL 同步到下游执行，并立刻返回，不会阻塞后续的 DML 执行。

Comment (sdojjy) [reply ticdc/ticdc-ddl.md]: fixed

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-ddl.md [hunk 2]
```diff
@@ -46,9 +46,20 @@ summary: 了解 TiCDC 支持同步的 DDL 和一些特殊情况
...
 ## DDL 同步注意事项
 
+### 创建和添加索引 DDL 的异步执行
+
+为了减小对 Changefeed 同步延迟的影响，如果下游是 TiDB，TiCDC 会异步执行创建和添加索引的 DDL 操作，即 TiCDC 将 `ADD INDEX` 和 `CREATE INDEX` DDL 同步到下游执行后，会立刻返回，而不会等待 DDL 操作完成。这样可以避免阻塞后续的 DML 执行。
+
+> **注意：**
+>
+> - 如果下游 DML 的执行依赖于未完成同步的索引，DML 可能会执行得很慢，进而影响 TiCDC 的同步延迟。
+> -在同步 DDL 到下游之前，如果 TiCDC 节点宕机或者下游有其他写操作，该 DDL 存在极低的失败概率，用户可自行检查。
```
Comment (qiancai) [inline ticdc/ticdc-ddl.md:56]:
```suggestion
> - 在同步 DDL 到下游之前，如果 TiCDC 节点宕机或者下游有其他写操作，该 DDL 存在极低的失败概率，用户可自行检查。
```

**Review by qiancai — APPROVED** [review]

File: ticdc/ticdc-ddl.md [hunk 3]
```diff
@@ -46,9 +46,20 @@ summary: 了解 TiCDC 支持同步的 DDL 和一些特殊情况
...
 ## DDL 同步注意事项
 
+### 创建和添加索引 DDL 的异步执行
+
+为了减小对 Changefeed 同步延迟的影响，如果下游是 TiDB，TiCDC 会异步执行创建和添加索引的 DDL 操作，即 TiCDC 将 `ADD INDEX` 和 `CREATE INDEX` DDL 同步到下游执行后，会立刻返回，而不会等待 DDL 操作完成。这样可以避免阻塞后续的 DML 执行。
+
+> **注意：**
+>
+> - 如果下游 DML 的执行依赖于未完成同步的索引，DML 可能会执行得很慢，进而影响 TiCDC 的同步延迟。
+> - 在同步 DDL 到下游之前，如果 TiCDC 节点宕机或者下游有其他写操作，该 DDL 存在极低的失败概率，用户可自行检查。
```
Comment (hfxsd) [inline ticdc/ticdc-ddl.md:56]:
```suggestion
> - 在同步 DDL 到下游之前，如果 TiCDC 节点宕机或者下游有其他写操作，该 DDL 存在极低的失败概率，你可以自行检查。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**